test.sh

```diff
@@ -0,0 +1,5 @@
+if git diff-index --quiet HEAD --; then
```

Comment:
???? :confused:

Reply:
I have no idea where these are coming from.

> Here's a better `diff` that makes it clear this is also removing a lot of the verification from previous releases. 😕


We changed how we performed the verification.  We moved from using the keys to verifying with sha512.

Reply:
> The `Dockerfile` should be written to help mitigate interception attacks during build. Our requirements focus on three main objectives: verifying the source, verifying author, and verifying the content; these are respectively accomplished by the following: using https where possible; importing PGP keys with the full fingerprint in the `Dockerfile` to check signatures; embedding checksums directly in the `Dockerfile`. All three should be used when possible.
>
>\-https://github.com/docker-library/official-images/tree/4f5972b48997d8620b9e3a8dca95b2e9db10d153#image-build

Reply:
Some of us were unable to get the key servers to return the keys needed.  This has been an issue for a few months, so we decided to remove them and add the sha512.  I can add them back.

Reply:
Thanks, using keys is definitely better verification.

Yeah, unfortunatetly the `sks-keyservers.net` service has been deprecated (GDPR takedown drama). So you'll have to use alternatives like MIT and `keyserver.ubuntu.com`. If you can get all the keys verified on `keys.openpgp.org` they have really good reliability.

Reply:
@yosifkit , thank you for the tip keyserver.ubuntu.com works great.  However, we have an issue with the 9.0 keys on our side.  So, we will meet you in the middle:  I added keys to 8.0, left them out for 9.0.  I will get on getting the keys fixed on our end.

Watch for a new PR.  I will close this one when the next is ready.